paddle/go/pserver/client.go

```diff
@@ -0,0 +1,61 @@
+package pserver
+
+// ElementType is the type of elements of a Parameter.
+type ElementType int
+
+// Supported element types
+const (
+	Int32 ElementType = iota
+	UInt32
+	Int64
+	UInt64
+	Float32
+	Float64
+)
+
+type Parameter struct {
+	Name        string
+	ElementType ElementType
+	Content     []byte
+}
+
```

Comment:
do we need to define  the ParameterBlock type(partitioned Parameter ) and put these things in a Constant header file? These things  would be a shared concept in many files.